### ISSUE TITLE
ghosting problem in 3 rows and 6 columns 64 by 64 P3 Led modules

### ISSUE THREAD

**Issue by hemantjha28** [hzeller/rpi-rgb-led-matrix#703]
Hi,

I am facing a very peculiar problem. It would be great of you can help me find out the root cause and the possible solution.

I have a led screen with 3 rows and 6 columns 64 by 64 P3 Led modules. I am using active adapter board as suggested by you.

The screen works absolutely fine if I use only 12 modules in 3 row by 4 column configuration. But the moment we arrange it in a 18 module 3 by 6 configuration the ghosting problem appears on the extreme end.

We have tried using official pi power supply but that is not working with 18 modules. Then I powered pi with with LM256 DC Dc buck convertor so that I can vary the voltage. This is where i found out the following issue.  If I supply pi3 b+ a input voltage of 5.18 volts the ghosting problem occurs while if i increase the input splly voltage to pi to around 6 volts the ghosting reduces drastically although there is still some ghosting. I didnt try increasing the voltage above 6 as it way burn by pi.

IMG 1 is the result with input voltage 5.18
IMG 2 is the result with input voltage 6
IMG3 shows you the connections.

It looks like even with active adapters the output of 5v diminishes with the increase in the number of modules in a row. Is there a way to address this problem.

The current consumed by pi3 b+ was around 0.5 amps.

I am using a 40 amps power supply but as I am showing simple text the current drawn from the power supply is not more than 3 amps.

I am using the active adapter broad provided by you.

Thanks a lot in advance.

Hemant
![img1](https://user-images.githubusercontent.com/16458265/47198428-b37b9080-d389-11e8-878b-e3bae435ee28.jpg)
![img2](https://user-images.githubusercontent.com/16458265/47198429-b37b9080-d389-11e8-8c6b-e1f6c89ed433.jpg)
![img3](https://user-images.githubusercontent.com/16458265/47198430-b4142700-d389-11e8-9906-b2f485a46834.jpg)

**Comment by davemaster** [comment]
Greetings,

Try using --led-slowdown-gpio=2 as You're using a RPi 3.

Best regards

**Comment by hemantjha28** [comment]
Thanks a lot David,

It worked. 

Is it for all RPi 3 we have to use --led-slowdown-gpio=2 irrespective of the screen size?

Regards

**Comment by davemaster** [comment]
Maybe with a bigger screen, You can use --led-slowdown-gpio=1 with the raspberry pi 3.

Please, take a picture of your screen that worked. By the way, did You use python to get that output on you screen.

Thanks in advance

**Comment by hemantjha28** [comment]
Yes I was using python. Sure I will send you the image in couple of days as the screen is at a different place.

**Comment by hzeller** [comment]
(disregard the above mention in the last commit. I meant #708 not 703)

**Comment by marcmerlin** [comment]
With apologies to everyone, this project is doing a one time cleanup of 
all the bugs that have not been addressed and closed in the last 10 years.

If you filed an issue to ask for help, please do not re-open this one or a new
bug, but go here instead: https://rpi-rgb-led-matrix.discourse.group/

Please be mindful that this project does not have any full time maintainers
or anyone paid. You should not expect personal help for your own projects or
answers to programming questions but you can ask for help with panels that
aren't working, setup questions, or anything that may be a bug here:
https://rpi-rgb-led-matrix.discourse.group/
If you need programming help and are willing to pay someone for it, please
state so in your message, and someone may be interested.

If you did report an actual bug that is still not fixed today, please accept
our apologies for this being mass closed, you can absolutely send a followup
and re-open the issue, or open a new up to date one as applicable.
Note that we very much welcome help, so if you can help fix your bug, you will
be a superstar :)

If your bug was linked to a pull request that was not resolved, please update
the pull request and the issue, and we will do our best to review them. We do
thank you for your undertanding that we have very limited free time, so any
extra work you do along with your bug report or PR request will be very much
appreciated and improve the odds that they will be looked at.

If you want to help contribute as a python or C# or any other language bindings,
you are also a superstar, please state so in the issue and we'll do our best to
give you the access and reviews you need.

For anything related to PWM panel support, please make sure you read README and
then refer to https://github.com/hzeller/rpi-rgb-led-matrix/issues/466 . 
Please do not comment if you're haven't read everything and/or if you're just
asking when your panel is going to be supported. Those bugs are if you're 
able to contribute to the effort.

For rPI5: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1603

For anything python, please link your bug to: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1828
For anything C#, please link your bug to: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1828
